data/pilots/galactic-empire/gauntlet-fighter.json

```diff
@@ -107,7 +107,6 @@
         "Configuration"
```

Comment:
Needs an illicit slot before modification, and I can't just make the code suggestion because GitHub is absolutely deranged.